lib/remote/httpserverconnection.cpp

```diff
@@ -541,13 +543,16 @@ void HttpServerConnection::ProcessMessages(boost::asio::yield_context yc)
 
 			Log logMsg (LogInformation, "HttpServerConnection");
 
-			logMsg << "Request: " << request.method_string() << ' ' << request.target()
+			logMsg << "Request " << request.method_string() << ' ' << request.target()
 				<< " (from " << m_PeerAddress
 				<< "), user: " << (authenticatedUser ? authenticatedUser->GetName() : "<unauthenticated>")
```

Comment:
Please remove the closing brace here:
```bash
(from [::ffff:127.0.0.1]:58460), user: root, agent: Icinga/DebugConsole/v2.14.0-91-g6c0359867, status: OK)
```

Reply:
Ah, yes. Git blame to the rescue: https://github.com/Icinga/icinga2/commit/5c32a5a7dcd220598d36b2b47e745d14c23edb93#diff-c9a6a6ab9c9b4c74648708955638eb06c64e93532ad595aa3eb58f879abb3cd6R217